frontend/css/home_view.css

```diff
@@ -27,6 +88,10 @@ body {
     text-align: center;
 }
 
+.joint-area {
+    margin-top: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=6343113)